config/settings.yml

```
@@ -1,3 +1,7 @@
 defaults: &defaults
   help_link: "http://www.railstutorial.org/help"
   book_link: "http://www.railstutorial.org/book"
+  user:
+    MAX_NAME_LENGTH: 50
+    MAX_EMAIL_LENGTH: 255
+    BIRTHDAY_YEAR_LIMIT: 100

```

```suggestion
    max_name_length: 50
    ...
```
Đưa vào settings rồi thì đặt tên biến bình thường thôi, còn biến const trong model mới cần viết hoa